.vscode/launch.json

```diff
@@ -5,11 +5,12 @@
       "type": "node",
       "request": "launch",
       "name": "Next.js Node Debug",
-      "runtimeExecutable": "${workspaceFolder}/node_modules/next/dist/bin/next",
+      "runtimeExecutable": "${userHome}/.yarn/bin/yarn",
```

Comment:
Why do we need this? What it is solving?

Reply:
Oh I just found it more effective using yarn than the script that dev is calling, but I can revert, not a needed change for the PR